### PR TITLE
Replace `docker ssh` with `docker-machine ssh`

### PR DESCRIPTION
**Pull request by crittermike**

### CODE REVIEW

**Review by grayside — DISMISSED** [review]
The problem with this is that the /data/projectname convention comes from the generator. If we create some generator-oriented docs, or document this as a convention we're good.

**Comment by crittermike** [comment]
Well the more urgent issue IMO is that `docker ssh` isn't a real command. Seems like the docs originally meant to say `docker-machine ssh` which is the main change I put in here.

**Comment by grayside** [comment]
Oh, I didn't see that there was already a command for this in place.

File: docs/common-tasks/stopping-containers-and-cleanup.md [hunk 1]
```diff
@@ -22,4 +22,9 @@ If you only want to clean up project specific stopped containers, you can run: `
 
 When you are finished with a project, if you used any persistent data storage you'll want to run a command to clean it 
 up. The exact directory to request removal from will depend on your project (see suggested directory naming guidelines in 
-[Key Concepts](../project-setup/key-concepts.md#persistent-data-volume)): `docker ssh dev sudo rm -rf /data/[project]/`
+[Key Concepts](../project-setup/key-concepts.md#persistent-data-volume)): 
+
+```
+docker-machine ssh dev
+sudo rm -rf /data/[project]/
```
Comment (grayside) [inline docs/common-tasks/stopping-containers-and-cleanup.md:29]: Would this be cleaner as a single command?

`docker-machine ssh dev sudo rm -Rf /data/[projectname]/`

**Review by grayside — COMMENTED** [review]
Let's go ahead and get the minimal fix in, I'll file a bug issue.

**Comment by crittermike** [comment]
I personally like it better as two commands because I think people will
want to look around a little bit before they know exactly what they want to
delete, and having it as one command makes it slightly more confusing if
you want to do that.

**Review by grayside — APPROVED** [review]
Didn't realize this was still languishing.

**Pull request merged**